dm/ctl/common/config.go

```diff
@@ -95,6 +99,15 @@ func (c *Config) Parse(arguments []string) (finish bool, err error) {
 		return true, nil
 	}
 
+	if len(c.decrypt) > 0 {
+		plaintext, err1 := utils.Decrypt(c.decrypt)
+		if err1 != nil {
+			return true, err1
+		}
+		fmt.Println(plaintext)
```

Comment:
this line should be deleted

Reply:
Perhaps we are shouldn't delete these lines, because the `Decrypt` method is called instead of `DecryptOrPlaintext` in L103 , the function may return a error